src/frontend/e2e/tests/sample.spec.ts

```diff
@@ -30,12 +34,21 @@ test.describe("Samples page tests", () => {
   });
 
   test("Should verify sample data", async ({ page }: { page: Page }) => {
+    await page.goto(process.env.BASEURL+"data/samples" as string);
```

Comment:
does this have to be cast as string?

Reply:
No it does have to be; just making TS happy

Reply:
what type is `process.env.BASEURL+"data/samples"` if it's not already a string? 😕

Reply:
ok i'll ignore for now, would be nice moving forwards to not make type assertions like this if possible 😄

Reply:
Let's fix this one too before we merge. 
We can tell TS what types are in process.env, so we don't need the assertions
https://stackoverflow.com/questions/45194598/using-process-env-in-typescript